akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/RunningMeeting.scala

```diff
@@ -19,9 +21,11 @@ object RunningMeeting {
 
 class RunningMeeting(val props: DefaultProps, outGW: OutMessageGateway,
                      eventBus: InternalEventBus)(implicit val context: ActorContext) {
-
+  val objectMapper: ObjectMapper = new ObjectMapper();
+  objectMapper.registerModule(new DefaultScalaModule())

```

Comment:
Rabbit suggeston makes sense.

I would also leave `RunningMeeting` cleaner.
It's better to call `private val plugins = PluginModel.createPluginModelFromJson(props.pluginProp)` and make all the process inside `PluginModel`.